gradle/libs.versions.toml

```diff
@@ -4,16 +4,16 @@ androidxannotations = "1.7.1"
 androidxactivity = "1.8.2"
 androidxpreference = "1.2.1"
 
-compose = "2023.10.01"
-composeMaterial3 = "1.1.2"
-compose-compiler = "1.5.6"
+compose = "2024.01.00"
+composeMaterial3 = "1.2.0-rc01"
```

Comment:
I can downgrade this one if necessary, but `1.1.2` has some major crash-inducing animation bugs at the moment.

Reply:
Is there any 1.1.3...? I don't want to use RCs if i can avoid that

Reply:
Not yet unfortunately.